CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+3.2.1 (2016-06-16)
+---------------------
+- Fido is fork-safe (reactor is not initialized at fido import time).
```

Comment:
you are right, I mentioned this in the description of this PR but I got carried away in comments, fixing